spec/syntax.md

```diff
@@ -71,6 +72,8 @@ No whitespace is allowed between components of an `InterpolatedUnquotedUrlConten
 [identifier code point]: https://drafts.csswg.org/css-syntax-3/#identifier-code-point
 [escape]: https://drafts.csswg.org/css-syntax-3/#escape-diagram
 
+### `FunctionExpression`
```

Comment:
Is there a link missing in this section pointing to spec/functions.md? Or is this not meant to be added here?

Reply:
Oh, this is just a leftover from my previous structure. Removed.